pkg/agent/apiserver/handlers/networkpolicy/handler.go

```diff
@@ -62,6 +62,7 @@ var mapToNetworkPolicyType = map[string]cpv1beta.NetworkPolicyType{
 	"NP":    cpv1beta.K8sNetworkPolicy,
 	"K8SNP": cpv1beta.K8sNetworkPolicy,
 	"ACNP":  cpv1beta.AntreaClusterNetworkPolicy,
+	"ANNP":  cpv1beta.AntreaNetworkPolicy,
 	"ANP":   cpv1beta.AntreaNetworkPolicy,
```

Comment:
Keeping the "ANP" here for AntreaNetworkPolicy, and in `antctl` util "ANP" is removed for AntreaNetworkPolicy, or until AdminNetworkPolicy is supported. Let me know your thoughts, thanks.